modules/javafx.graphics/src/main/native-glass/mac/GlassKey.m

```diff
@@ -178,6 +180,166 @@
 };
 static const int gKeyMapSize = sizeof(gKeyMap) / sizeof(struct KeyMapEntry);
 
+static BOOL macKeyCodeIsLayoutSensitive(unsigned int keyCode)
+{
+    // Mac key codes that generate different characters based on the keyboard layout
+    // lie in two ranges with a few exceptions.
+    switch (keyCode)
+    {
+        case 0x24: // Enter
+        case 0x30: // Tab
+        case 0x31: // Space
+            return NO;
+    }
+
+    if (keyCode >= 0x00 && keyCode <= 0x32)
```

Comment:
No real reason, it was probably just easier to write it like this when I was originally going through the table and sorting out the ranges. I do prefer separating out conditionals when I can since it makes it easier to set debug breakpoints (not that that matters here).

Reply:
okay, no problem